pkg/backup/actions/csi/pvc_action.go

```diff
@@ -50,6 +57,14 @@ import (
 	kubeutil "github.com/vmware-tanzu/velero/pkg/util/kube"
 )
 
+// Finalizer constants
+const (
+	VGSKind                                 = "VolumeGroupSnapshot"
+	VGSAPIVersion                           = "groupsnapshot.storage.k8s.io/v1beta1"
+	VolumeSnapshotFinalizerGroupProtection  = "snapshot.storage.kubernetes.io/volumesnapshot-in-group-protection"
```

Comment:
Please reference to the external-snapshotter original definition.
https://github.com/kubernetes-csi/external-snapshotter/blob/6b2feaaf7fc3d6d0fe21029d1032ea0eee2081e9/pkg/utils/util.go#L86